packages/core/stories/tutorials/combo-charts.ts

```diff
@@ -0,0 +1,114 @@
+import marked from "marked";
+
+export const comboChartTutorial = {
+	name: "Combo Chart Tutorial",
+	content: marked(`
+# Combo charts
+Combo charts allow users to visually compare complex datasets by superimposing different graphing components within one chart.
+Combination charts **may or may not** use dual axes although it is common that they do. For more information on dual axes with **carbon-charts**, please see [this tutorial](https://carbon-design-system.github.io/carbon-charts/?path=/story/tutorials--dual-axes).
+
+*Note: \`ComboChart\` extends axis chart functionality and should only comprise of graphs/components that use axes.*
+
+____
+
+### Using the carbon-charts defaults
+
+There is support for using pre-existing carbon-charts graphs and combining them together within a \`ComboChart\`.
+The charts that can be seamlessly combined are all different **axis charts only**. This is the **reccommended way to use \`ComboCharts\`**.
+
+Supported \`ChartTypes\` to combine:
+- \`LineChart\`
+- \`ScatterChart\`
+- \`StackedScatterChart\`
+- \`AreaChart\`
+- \`StackedAreaChart\`
+- \`BarChart\`
+- \`GroupedBarChart\`
+- \`StackedBarChart\`
+
+Below is an example of combining a carbon-charts \`AreaChart\` with a \`LineChart\`.
+
+\`\`\`
+const verySimpleComboData = [
+    { group: "Dataset 1", value: 650, date: "Tuesday" },
+	{ group: "Dataset 2", value: 296, date: "Tuesday" }
+	...more data
+];
+
+const comboOptions = {
+	...otherOptions // global options apply to all the charts listed in chartTypes (where relevant)
+	axes: {...}
+	comboChartTypes: [ // takes an array of objects
+		{
+			type: "area" // use a valid ChartType from enums
```

Comment:
`// use a valid ChartType from enums` can we please add the proper import to the code sample here?